ui/static/app/http-client.ts

```diff
@@ -12,14 +12,12 @@ export class HttpClient {
   constructor(
     private http: Http,
     private authService: AuthService
-  ) {
-    this.apiToken = this.authService.getToken();
-  }
+  ) { }
 
   getAuthorizationHeader(): Headers {
     let headers: Headers = new Headers();
     headers.append("Content-Type", "application/json");
-    headers.append("Authorization", `Token ${this.apiToken}`);
+    headers.append("Authorization", `Token ${this.authService.getToken()}`);
```

Comment:
Please no. In this case, before every requesta to api there will be additional request for user token, am i right?